pkg/operator/bootstrap.go

```diff
@@ -94,3 +126,50 @@ func (c *Controller) Bootstrap() error {
 
 	return nil
 }
+
+func (c *Controller) createPVC(accessMode corev1.PersistentVolumeAccessMode) error {
+	claimName := pvc.PVCImageRegistryName
+
+	// Check that the claim does not exist before creating it.
+	claim, err := c.clients.Core.PersistentVolumeClaims(defaults.ImageRegistryOperatorNamespace).Get(claimName, metav1.GetOptions{})
+	if err == nil {
+		if !pvc.PVCIsCreatedByOperator(claim) {
+			err = fmt.Errorf("could not create this PVC, it already exists and is not owned by the operator")
+			return err
+		}
+		// PVC is already created, no bootstrapping is required
+		return nil
+	} else if errors.IsNotFound(err) {
+		// if PVC with this name doesn't exist, create new PVC
+		storageClassName := "standard"
```

Comment:
Is it a special name for our OpenShift on OpenStack?

Reply:
yes, it's kinda hardcoded in kubernetes: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/storage-class/openstack/default.yaml#L4